src/CRM/CivixBundle/Resources/views/Code/mkdocs.yml.php

```diff
@@ -0,0 +1,21 @@
+site_name: <?php echo "$fullName\n"; ?>
+repo_url: https://lab.civicrm.org/extensions/FIXME
+theme: material
+
+pages:
```

Comment:
```suggestion
pages:
```

The pages key is deprecated. It won't work. This should use `nav:`